### PR TITLE
cartopy

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @pelson. I hope this is OK :stuck_out_tongue_winking_eye:

### CODE REVIEW

**Comment by pelson** [comment]
Hehe. Cheeky :smile:
Yes, this is awesome. Thanks @ocefpaf!

**Comment by ocefpaf** [comment]
I just submitted Proj.4, I suspect that will resolve the AppVeyor failure.

```shell
proj.lib(PJ_bipc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_aeqd.obj) : error LNK2019: unresolved external symbol __imp_hypot referenced in function e_forward
proj.lib(PJ_geos.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(proj_etmerc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_eqdc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_bonne.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_aea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_lcc.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_laea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_gnom.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_sterea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_ortho.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_oea.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_mod_ster.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_sconics.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_stere.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_nsper.obj) : error LNK2001: unresolved external symbol __imp_hypot
proj.lib(PJ_tpeqd.obj) : error LNK2001: unresolved external symbol __imp_hypot
build\lib.win-amd64-3.4\cartopy\_crs.pyd : fatal error LNK1120: 1 unresolved externals
error: command 'c:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\VC\\Bin\\amd64\\link.exe' failed with exit status 1120
```

**Comment by ocefpaf** [comment]
We are still getting Proj.4 from the default channel because the version and build number are the same :unamused: 

`proj.4-4.9.1               |                0         1.4 MB  defaults`

As we don't have a long term solution for this I suggest I bump the build number of our Proj.4. Is that OK @pelson?

**Comment by pelson** [comment]
This is a problem with the default build, right? Pinging @mcg1969 to ensure you have awareness of it. We could come up with a hacky solution (such as bumping the build number) but this is a really great example of having community recipes which are ahead of those on defaults.

**Comment by msarahan** [comment]
Yeah, until channel priority is a thing, I am not sure how to work around this.  Depending on build number is bad because it is so ambiguous.

Side note: I think it's good practice to set build number to 0 when merging a new recipe here, but that really complicates this issue.

**Comment by ocefpaf** [comment]
`proj.4` for py3k did not make into the channel. Investigating now...

File: recipes/cartopy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+source:
+    git_url: https://github.com/SciTools/cartopy.git
+    git_tag: v{{ version }}
+    patches:
+        - cartopy.win.patch  # [win]
+
+build:
+    number: 0
+    skip: True  # [py35 and win]
```
Comment (ocefpaf) [inline recipes/cartopy/meta.yaml]: Ping @pelson: how about a new release :wink:

**Comment by ocefpaf** [comment]
@pelson our version of proj.4 did the trick. We have green!

**Pull request merged**